hack/jenkins/e2e.sh

```diff
@@ -206,6 +207,20 @@ case ${JOB_NAME} in
     NUM_MINIONS="6"
     ;;
 
+  # Runs all non-flaky tests on AWS in parallel.
+  kubernetes-e2e-aws-parallel)
+    : ${E2E_CLUSTER_NAME:="jenkins-aws-e2e-parallel"}
+    : ${E2E_NETWORK:="e2e-parallel"}
+    : ${GINKGO_PARALLEL:="y"}
+    : ${GINKGO_TEST_ARGS:="--ginkgo.skip=$(join_regex_allow_empty \
+          ${GCE_DEFAULT_SKIP_TESTS[@]:+${GCE_DEFAULT_SKIP_TESTS[@]}} \
```

Comment:
if we end up sharing these lists, maybe we'll want to drop "GCE" from the name at some point...